modules/tinymce/src/themes/silver/main/ts/api/Settings.ts

```diff
@@ -135,6 +134,22 @@ export interface ToolbarGroupSetting {
 
 const getToolbar = (editor: Editor): Array<string | ToolbarGroupSetting> | string | boolean => editor.getParam('toolbar', true);
 
+const getFilePickerCallback = (editor: Editor) => editor.getParam('file_picker_callback');
+
+const getFilePickerTypes = (editor: Editor) => editor.getParam('file_picker_types');
+
+const getFileBrowserCallbackTypes = (editor: Editor) => editor.getParam('file_browser_callback_types');
+
+const noTypeHeadUrls = (editor: Editor) => editor.getParam('typeahead_urls') === false;
```

Comment:
Typo here and `Typeahead` is one word.

Reply:
Oh, "Type ahead", not "Type head". I completely missed that.